iOSClient/Utility/CCUtility.m

```diff
@@ -678,8 +679,11 @@ + (NSInteger)getChunkSize
     }
 }
 
+/// In megabytes (MB)
 + (void)setChunkSize:(NSInteger)size
 {
+    if (size < 10) size = 0;
```

Comment:
I think the size should still be default to the minimum supported value of 5MB then if configured to be > 0.

Maybe there is some byte calculation in another place where the 5 MB -> bytes generates the wrong value?

Reply:
You mentioned the min server size is 5 MiB, not MB. So maybe we will set it to this, or keep it to 10MB as minimum. We are also adding auto chunking, so we'll see how we'll implement this